add bio

kvenkman opened the pull request.

kvenkman committed:
add bio

scottyhq commented:
@jomey I'm not sure what the issue is here, but in the preview Karthik and Kehan's profiles seem to be getting merged ?!

jomey commented on team/karthik-venkataramani.yaml:
```diff
@@ -0,0 +1,19 @@
+      link: https://www.linkedin.com/in/karthikvenkataramani/
+  user_groups:
+    # From https://escience.washington.edu/using-data-science/hackweeks/planning/hackweek-roles
+    - Helper
```
@scottyhq - Looks like we have an issue with the YAML here.
Should look like this:
```
  roles:
    - Helper
```

scottyhq replied:
I don't think that's it

https://github.com/uwhackweek/event-page-2024/blob/099ce079e3ce59603b8e921567796408e3be561d/team/template.yaml#L20-L23

maybe some special character situation or subtle thing with the way individual files get merged...

jomey replied:
> I don't think that's it
> 
> https://github.com/uwhackweek/event-page-2024/blob/099ce079e3ce59603b8e921567796408e3be561d/team/template.yaml#L20-L23
> 
> maybe some special character situation or subtle thing with the way individual files get merged...

Oh .. yeah. That can't be it.
Let me have another look

jomey committed:
Team - Fix misteriously corrupt line ending

One profile seemed to have an unrecognized newline at the end of their
profile, causing a merge with the previous

scottyhq commented:
@kvenkman out of curiosity what code editor and OS were you using to create your file?
